homeassistant/loader.py

```diff
@@ -366,7 +366,7 @@ def resolve_from_root(
                 continue
 
             try:
-                manifest = json.loads(manifest_path.read_text())
+                manifest = json_loads(manifest_path.read_text())
             except ValueError as err:
```

Comment:
Should this catch `JSON_DECODE_EXCEPTIONS` ?

Reply:
Yeah that would be nicer as it would lower the long term maint burden

Reply:
615f0f38b1fad3105bcf84e7c87863e8cb67560a